### DISCUSSION TITLE
How I can scan Routers / Switches with Nuclei?

### DISCUSSION THREAD

**Discussion by maorkuriel** [projectdiscovery/nuclei#4534] [Q&A] [answered]
I saw that there is an Alpha that enables Nuclei to identify vulnerabilities in network devices like routers, switches, and firewalls.

Where can I find instructions on how to run these scans? 

Thanks in advance.

**Comment by olearycrew** [comment, marked as answer]
Can you provide more detail on exactly what you're looking for? There are a lot of network templates as well as HTTP templates designed to be run against network devices.

Have you checked out the Nuclei Templates repo? https://github.com/projectdiscovery/nuclei-templates
